pkg/kubelet/metrics/metrics.go

```diff
@@ -786,6 +789,15 @@ var (
 			StabilityLevel: metrics.ALPHA,
 		},
 	)
+
+	ImageGarbageCollectedTotal = metrics.NewCounter(
```

Comment:
In other areas of the code, I was told that we don't add metrics for alpha features.  

I would like to include this in alpha but not sure if this is usually what we do for alpha features.

Reply:
this isn't so much a metric for an alpha feature, but a metric that will be useful to inform how well an alpha feature is working compared to when the feature is turned off. it'll measure GCs independent of the feature being on

Reply:
That sounds good to me.